### DISCUSSION TITLE
Switch or option to disable 0w miners

### DISCUSSION THREAD

**Discussion by Bitofsin** [RainbowMiner/RainbowMiner#2811] [Ideas]
Would it be possible to add a switch or config option that will disable miners that benchmark and have zero watts. I have some miners running in profitability mode and it is auto selecting the zero watt algorithms as the most profitable. To fix this I have to login and manually disable all algos with zero watts on each rig.

**Comment by RainbowMiner** [comment]
Sorry, I found this a little late. But that option is possible, of course. I will add it.

**Reply by Bitofsin** [reply]
Thanks for getting back to me! No worries about the delay—I appreciate you adding the feature. I also ran into an issue where all miners crash immediately on benchmark with no error when using 12 of the same GPUs (1660). Is there anything I should try to fix this?

**Reply by RainbowMiner** [reply]
I have now added a new parameter to config.txt: "DisableZeroWattMiners"

**Comment by Bitofsin** [comment]
Sweet thank you
